setup.py

```diff
@@ -6,20 +6,34 @@
 with open('README.rst') as f:
     readme = f.read()
 
-with open('LICENSE') as f:
-    license = f.read()
+install_requires = [
+                    'hvac==0.2.17+git.b817da4',
+                    'requests',
+]
+
+dependency_links = [
+    'https://github.com/mracter/hvac/archive/b0811f0446760125baefea6383711035104e6df1.zip#egg=hvac-0.2.17+git.b817da4',
+]
 
 setup(
     name='vaultkeeper',
     version='0.0.1',
-    description='A daemon that fetches and renews '
-                + 'Vault credentials for Gunicorn-on-Docker'
-                + 'Django applications',
+    description='An agent that works with '
+                'vault-gatekeeper-mesos to '
+                'fetch and renew Vault credentials.',
```

Comment:
For multiline implicit concatenation, it's generally a good idea to surround the string with parens to visually indicate that it's a single entity:
```
description=('jhlkh shyek'
             ' alkjlu apofchpo adbhaue')
```